admin.rb

```diff
@@ -13,10 +13,10 @@ class Admin < Formula
 
   if OS.mac?
     url "#{base_url}/#{file_name}-darwin-amd64"
-    sha256 "a138a253b511aa9be49dc7b4e4380ef8351a4e96f66b86c32b62ad7af8728050"
+    sha256 "f4f94cda09c8fcd21b0a7c8adf0d57ee0e56c7ac026e2fd07d0954ff9abdbee7"
```

Comment:
I would like to see this being calculated by automation script

Reply:
@cardil has done that for the event plugin, which I think is going to become the standard. [slack discussion](https://knative.slack.com/archives/C02SFDXD72A/p1643207831189200)